src/js/providers/html5.js

```diff
@@ -668,7 +668,7 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
     };
 
     this.getQualityLevels = function() {
-        return _.map(_levels, level => qualityLevel(level));
+        return map(_levels, level => qualityLevel(level));
```

Comment:
Can we use `_levels.map` here and remove the import? If it's possible for `_levels` to not be an array, then return an empty array when `!_levels`. That would be cheaper than the import.